README.md

```diff
@@ -41,26 +41,31 @@ This requires a free FMOD account.
 
 ```toml
 [dependencies]
-bevy_fmod = { git = "https://github.com/Salzian/bevy_fmod.git", tag = "v0.0.1" }
+bevy_fmod = { git = "https://github.com/Salzian/bevy_fmod.git", tag = "v0.1.0" }
 ```
 
 This repository includes [examples](./examples) and a FMOD demo project.
 
 ## Versioning
 
-Version 0.1.0 would be the first usable MVP somewhat recommended for use.  
+Version 0.0.1 would be the first usable MVP somewhat recommended for use.  
 Version 1.0.0 would be a hypothetical stable release. But that's still a long way to go.
```

Comment:
Not true in the sense I initially intended to version this package, but could you just remove these two lines? Looking at it now, this "system" doesn't make much sense. There should only be 0.x and >=1.x. thx